### DISCUSSION TITLE
一次查多个timeseries有问题

### DISCUSSION THREAD

**Discussion by luanshidujun** [apache/iotdb#4542] [Q&A] [answered]
我从12.2版更新到12.3版
12.2版时，一次可以查1万个timeseries，但换了12.3版后就报
Too many paths in one query! Currently allowed max deduplicated path number is 1000. Please use slimit or adjust max_deduplicated_path_num in iotdb-engine.properties.
把这个配置项给成1万也没用，依然报上述错误，这个配置项的说明是就算把数值改大也不见得有效
# allowed max numbers of deduplicated path in one query
# it's just an advised value, the real limitation will be the smaller one between this and the one we calculated
max_deduplicated_path_num=10000
那问题是这个算出来的值究竟是怎么算的，我的服务器性能很强大，但依然不能一次查1万个timeseries，这样太影响效率了
![image](https://user-images.githubusercontent.com/37785928/143538120-6449992f-7f5c-40e1-ba3b-098707ff1427.png)

**Comment by luanshidujun** [comment]
我试过，我的服务器上只运行iotdb12.3版,其他程序全部关闭，无论我怎么改max_deduplicated_path_num参数，只要查询是超过1000个timeseries就报错

**Comment by HTHou** [comment, marked as answer]
> 我试过，我的服务器上只运行iotdb12.3版,其他程序全部关闭，无论我怎么改max_deduplicated_path_num参数，只要查询是超过1000个timeseries就报错

0.12.3版本有一个已知bug，必须将 chunk_timeseriesmeta_free_memory_proportion=1: 100:200:300:400 这条配置参数解注释，max_deduplicated_path_num 的调整才会生效。可以试试。


另外，0.12.4 版本会在近期快速发布，会包含修复这个bug的commit https://github.com/apache/iotdb/pull/4406 。

**Reply by lichanging** [reply]
this resolves my problem!

**Comment by luanshidujun** [comment]
> 

谢谢，我试了，可以的
